docs/data/introduction/support/support.md

```diff
@@ -33,3 +33,19 @@ Including your Order ID (or Support key) in the issue helps us prioritize the is
 1. MUI X Pro: maintainers give these issues more attention than the ones from the community.
 2. MUI X Premium: The same as MUI X Pro, but with higher priority.
 3. MUI X Priority Support add-on (coming soon): Provides a 24h SLA for the first answer.
+
+### Long-term support (LTS)
+
+We roll bug fixes, performance enhancements, and other improvements into new releases, however, MUI remains committed to delivering security updates and addressing regressions for the immediate predecessor of the current major version.
+
+This includes tackling issues arising from external factors like browser upgrades or changes to upstream dependencies.
+
+### Supported versions
+
+- MUI X v7: ✅ Pre-release (Continuous support - Stable release: March 2024)
+- MUI X v6: ✅ Stable major (Continuous support)
+- MUI X v5: ⚠️ Long term support (Guaranteed Support for security issues and regressions).
+- MUI X v4: 🅧 No longer supported.
+- MUI X v3: 🅧 Never existed.
+- MUI X v2: 🅧 Never existed.
+- MUI X v1: 🅧 Never existed.
```

Comment:
IMHO I would not mention those at all ... especially since the `v4` is no longer supported

Reply:
Yeah, that's a close call, I was also uncertain if I should mention those.
I did mainly to be more compatible with the list at:
https://mui.com/versions/#supported-versions

Reply:
I got that ... IMO things that never existed in the first place can go unmentioned.
It's a bit different to the list in the link. There we do actually have versions v0.x, v1 and v3. For some reason only v2 got skipped